packages/client-sdk-nodejs-compression/package.json

```diff
@@ -48,8 +48,7 @@
   },
   "dependencies": {
     "@gomomento/generated-types": "0.107.4",
-    "@gomomento/sdk": "file:../client-sdk-nodejs",
-    "@mongodb-js/zstd": "1.2.0"
+    "@gomomento/sdk": "file:../client-sdk-nodejs"
```

Comment:
Since zlib is part of the node runtime, does this whole package need to be an external dependency then? Couldn't it be included with the nodejs client?

Reply:
we talked about this in the team meeting the other day. for now i'd prefer that users need to stop and consider options, so i like it being separate.